website/src/pages/ceros-microcontent.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../components/Layout'
+
+const CerosMicroContent: FunctionComponent<PageProps> = props => (
+    <Layout location={props.location}>
+        <div className="container py-3">
+            <div
+                style={{
+                    position: 'relative',
+                    width: 'auto',
+                    padding: '0 0 56.25%',
```

Comment:
Same question here, `56.25%`?

Reply:
Part of the embedded test code given. Reference in ticket